shimtest/mockstub.go

```diff
@@ -604,3 +631,12 @@ func getFuncArgs(bytes [][]byte) (string, []string) {
 	}
 	return function, args
 }
+
+// helper
+func marshalOrError(msg proto.Message) ([]byte, error) {
+	bytes, err := proto.Marshal(msg)
+	if err != nil {
+		return nil, err
+	}
+	return bytes, nil
+}
```

Comment:
This really isn't adding any value. The caller is simply calling this and handling the error instead of calling `proto.Marshal`. I can literally change `marshalOrError` to `proto.Marshal` and there would be no behavior changes.

Please remove this.